docs/conf.py

```diff
@@ -58,9 +58,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '5.0'
+version = '5.2'
```

Comment:
Please change this to 5.1.2 for now. Once this is in I'll update the tag per #356.